scripts/docker-init-container.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Did you not use `#!/usr/bin/env bash` for security reasons here?

Reply:
> Did you not use `#!/usr/bin/env bash` for security reasons here?

@AntoineRondelet  Changed to use env.